README.md

```diff
@@ -7,128 +7,33 @@
 
 > Project Management
 
-This is a repository for general ipfs project management.
+This is a repository for general IPFS project management.
 
-#### Please don't open issues here unless you know what you're doing.
+tl:dr; We have a **weekly, all-hands call**. The time and date is announced in [the issues](). Propose agenda items in the etherpad linked in that issue!
```

Comment:
- (not sure of typo or intentionally left blank) fill link to https://github.com/ipfs/pm/issues